--- start.ini	2012-02-02 15:53:30.702002593 +1100
+++ ../start.ini	2012-02-02 15:51:22.052002535 +1100
@@ -46,7 +46,7 @@
 # for a full listing do
 #   java -jar start.jar --list-options
 #-----------------------------------------------------------
-OPTIONS=Server,jsp,jmx,resources,websocket,ext
+OPTIONS=Server,jsp,jmx,resources,websocket,ext,jta,plus,jdbc,annotations
 #-----------------------------------------------------------
 
 
@@ -55,13 +55,16 @@
 # For a full list of available configuration files do
 #   java -jar start.jar --help
 #-----------------------------------------------------------
-#etc/jetty-jmx.xml
+etc/jetty-jmx.xml
 etc/jetty.xml
+etc/jetty-hightide.xml
 # etc/jetty-ssl.xml
 # etc/jetty-requestlog.xml
 etc/jetty-deploy.xml
+etc/jetty-plus.xml
 #etc/jetty-overlay.xml
 etc/jetty-webapps.xml
 etc/jetty-contexts.xml
 etc/jetty-testrealm.xml
+etc/jetty-jaas.xml
 #===========================================================
